packages/next-app/tailwind.config.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
         ...blackA,
         ...slateDark,
         ...blueDark,
+        ...violet,
```

Comment:
any reason to add another color?

Reply:
it's the check box "checked" color, but we can change it